### ISSUE TITLE
Automate PDP-11 stuffing

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1106]
The PDP-11s connected through 10-11 would normally be running continuously.  So there's no provision for stuffing them when ITS boots.

The scenario when using emulators is a bit different.  You would normally start all machines, and then you have to manually stuff all the 11s that you want to work with.

For the benefit of new users, it would be good if this could be automated.  Maybe a new RAKASH STUFF that creates inferior STUFF jobs for each PDP-11.

**Comment by larsbrinkhoff** [comment]
I propose that perhaps the simplest solution would be to have a DDT xfile script.

It could be something like this:

```
:stuff tv
:self
mumble/ -1
:forget
:job sys
$$^something
tt11p/1
:forget
```

If this were, say, SYSTEM; SYSTEM LOGIN, all you need to do is login as SYSTEM.

**Comment by larsbrinkhoff** [comment]
Fixed by #1167.
